moveit_simple/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 2.8.3)
 project(moveit_simple)
 find_package(catkin REQUIRED COMPONENTS
   roscpp
+  rosparam_handler
+  dynamic_reconfigure
```

Comment:
Do you need to depend on both rosparam_handler and dynamic_reconfigure?  One should include the other.

Reply:
I initially ran the tests with just dynamic_reconfigure but later ported to rosparam_handler.. I will remove the necessary dependencies.

Reply:
yes, so apparently dynamic_reconfigure needs to be explicitly added otherwise we get dependency issues.